docs/getting-started.md

```diff
@@ -6,6 +6,8 @@ However, if you want to integrate containerd into your project we have an easy t
 
 In this guide we will pull and run a redis server with containerd using the client package.
 We will assume that you are running a modern linux host for this example with a compatible build of `runc`.
+You can download one of the latest builds for runc on the [Github release](https://github.com/opencontainers/runc/releases) page and make sure runc installation path is registered in the `$PATH`.
+You also need Go version 1.8.x or above to follow this guide.
```

Comment:
Actually, they should use the version stated in [`RUNC.md`](https://github.com/containerd/containerd/blob/master/RUNC.md)